Makefile.am

```diff
@@ -818,6 +818,26 @@ endif
 
 check: $(CHECK_DEPS)
 
+artifacts: | install-image install-kernel
+
+# Get clear containers image path
+IMAGE_PATH := $(shell dirname $(CONTAINERS_IMG))
+install-image:
+	@if [ $(shell id -u) != 0 ]; then \
```

Comment:
Nit: this is using the string comparison operator rather than the numeric (`-ne`), but it still works so not really a problem.